build/build-sdk-images/node/gen.sh

```diff
@@ -24,15 +24,18 @@ sdk=/go/src/agones.dev/agones/proto/sdk
 googleapis=/go/src/agones.dev/agones/proto/googleapis
 gatewaygrpc=/go/src/agones.dev/agones/proto/grpc-gateway
 
-mkdir -p ./sdks/nodejs/lib/alpha
-
 cd /go/src/agones.dev/agones
 
+mkdir -p ./sdks/nodejs/lib/alpha
+mkdir -p ./sdks/nodejs/lib/beta
+
 grpc_tools_node_protoc -I ${googleapis} -I ${gatewaygrpc} -I ${sdk} --js_out=import_style=commonjs,binary:./sdks/nodejs/lib google/api/annotations.proto google/api/client.proto google/api/field_behavior.proto google/api/http.proto google/api/launch_stage.proto google/api/resource.proto protoc-gen-openapiv2/options/annotations.proto protoc-gen-openapiv2/options/openapiv2.proto
 grpc_tools_node_protoc -I ${googleapis} -I ${gatewaygrpc} -I ${sdk}/alpha --js_out=import_style=commonjs,binary:./sdks/nodejs/lib/alpha google/api/annotations.proto google/api/client.proto google/api/field_behavior.proto google/api/http.proto google/api/launch_stage.proto google/api/resource.proto protoc-gen-openapiv2/options/annotations.proto protoc-gen-openapiv2/options/openapiv2.proto
```

Comment:
Do we need all of these? Looks like the alpha proto has two imports:

https://github.com/googleforgames/agones/blob/1b06e945ec086d14ceffdb02ef818b768e206b6a/proto/sdk/alpha/alpha.proto#L20-L21

Reply:
This could be tested by removing the extraneous entries, regenerating and rerunning the node.js tests with `npm test`

Reply:
Actually I think it may need running the local server to fully test

Reply:
http.proto is used by annotations.proto, so need to check each file for dependencies, not just the top level
I can check in my branch